### DISCUSSION TITLE
Setting Up Server of Studies( Non Dicom Images)

### DISCUSSION THREAD

**Discussion by Ivan-E-Johnson** [Project-MONAI/MONAI#6587] [General]
Hello everyone,

I'm currently working on setting up a server so that multiple computers can access and annotate a single dataset composed of 4D ( C, H, W, D) Niffti images, in Monail Label. While there is plenty of documentation available for setting up servers for the DICOM image series, I couldn't find much information specifically focused on non-DICOM images.  If there isn't prebuilt functionality for this what specifications would a custom server need to provide to be able to interface with Monai Label and Slicer Plugin?

**Comment by tangy5** [comment]
Hi @Ivan-E-Johnson , monailabel can work with NIFTIs, nrrd, image formats.. etc..While most of our tutorials are with NIFTI and DICOM, as most public datasets are prepared. There are still challenges with 4D NIFTI images with 3D Slicer, such as BRATS. 

A custom model or app can adapt the 4D images for inference/training with monailabe, but we need to think about how to visualize them and interact with 3D Slicer. 

More details about multiple computers collaborative usage and authentications: https://github.com/Project-MONAI/MONAILabel/wiki/Integration-with-Keycloak
